### PR TITLE
Fetching data from redux for admin categories

### LINKED ISSUE #112: Connect the Categories in Admin section with backend
**Is your feature request related to a problem? Please describe.**
Feature request.
**Describe the solution you'd like**
Use redux.
**Additional context**
Are you working on this?
@Subhang23 Take this up. If you are unable to do it, please write it here so someone else can take this up.

### PR DESCRIPTION
**Pull request by Subhang23**
Fixes #112

Changes: Getting data for the table using redux

### CODE REVIEW

**Comment by ashwaniYDV** [comment]
@Subhang23 I've made categoryService Actions and Reducer and I'm working on it

File: src/redux/actions/adminCategoryActions.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import {FETCH_CATEGORY} from './type';
+
+export const fetchCategory =()=>{
+    return dispatch =>{
+        fetch('https://jsonplaceholder.typicode.com/posts')
```
Comment (harsh-jindal) [inline src/redux/actions/adminCategoryActions.js]: @Subhang23 Use axios and from what url are you fetching?

File: src/redux/actions/adminCategoryActions.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+            .then(response => response.json())
```
Comment (harsh-jindal) [inline src/redux/actions/adminCategoryActions.js]: Convert the response to json in one then itself.
two then's are Not required.

**Review by harsh-jindal — CHANGES_REQUESTED** [review]

**Comment by harsh-jindal** [comment]
@Subhang23 Also rename the adminCategoryReducer/Action to adminAction/Reducer

**Pull request closed without merging**